plugins/org.jboss.tools.openshift.express.core/src/org/jboss/tools/openshift/express/internal/core/server/OpenShiftServerExtendedProperties.java

```diff
@@ -24,6 +24,10 @@ public OpenShiftServerExtendedProperties(IAdaptable adaptable) {
 		super(adaptable);
 	}
 	
+	public boolean allowConvenienceEnhancements() {
```

Comment:
what does 'convenienceEnhancements" mean ? is that really our name for XML and File features ?